pkg/cmd/util/clientcmd/factory.go

```diff
@@ -510,12 +510,12 @@ func NewFactory(clientConfig kclientcmd.ClientConfig) *Factory {
 			if t.Spec.Paused {
 				return true, nil
 			}
-			t.Spec.Paused = true
+			body := []byte("{\"spec\":{\"paused\":true}}")
```

Comment:
@deads2k @liggitt agree, I can check if I can build something more minimal... I don't see an obvious way how to call set.CalculatePatches in here.